### ISSUE TITLE
Attendee/Speaker Directory

### ISSUE THREAD

**Issue by karthikb351** [hasgeek/android#5]
- List of all attendees (Who's agreed to share data)
 - List of speakers

**Comment by shreyas-satish** [comment]
Should this feature be restricted to people who registered for the conference?

**Comment by karthikb351** [comment]
Yeah, that's related to https://github.com/hasgeek/android/issues/7
